@ecomplus/storefront-components/src/js/AccountAddresses.js

```diff
@@ -0,0 +1,184 @@
+import { i18n } from '@ecomplus/utils'
+import {
+  i19edit,
+  i19newAddress,
+  i19noNumber,
+  i19remove
+} from '@ecomplus/i18n'
+import AddressForm from './../AddressForm.vue'
+
+const { sessionStorage } = window
+const storageKey = 'ecomCustomerAddress'
+
+export default {
+  name: 'AccountAddresses',
+
+  components: {
+    AddressForm
+  },
+
+  props: {
+    customer: {
+      type: Object,
+      default () {
+        return {}
+      }
+    },
+    zipCode: {
+      type: String
+    }
+  },
+
+  data () {
+    return {
+      canShowForm: false,
+      newAddress: false,
```

Comment:
Vamos tentar manter booleanos sempre com prefixo `is` ou `can`:
```suggestion
      isNewAddress: false,
```